packages/tools/nodeEditor/src/sharedComponents/color3LineComponent.tsx

```diff
@@ -75,19 +75,7 @@ export class Color3LineComponent extends React.Component<IColor3LineComponentPro
     }
 
     copyToClipboard() {
-        const element = document.createElement("div");
-        element.textContent = this.getCurrentColor().toHexString();
-        document.body.appendChild(element);
-
-        if (window.getSelection) {
-            const range = document.createRange();
-            range.selectNode(element);
-            window.getSelection()!.removeAllRanges();
-            window.getSelection()!.addRange(range);
-        }
-
-        document.execCommand("copy");
-        element.remove();
+        navigator.clipboard.writeText(this.getCurrentColor().toHexString());

```

Comment:
if you can use feature detection it will be the safest. i.e. check that `navigator.clipboard` exists and that it has the writeText function.